api/gateway.proto

```diff
@@ -244,7 +259,13 @@ message GatewayAntenna {
   float gain = 1;
   // location is the antenna's location.
   Location location = 2 [(gogoproto.nullable) = false, (validate.rules).message.required = true];
-  map<string,string> attributes = 3 [(validate.rules).map.keys.string = {pattern: "^[a-z0-9](?:[-]?[a-z0-9]){2,}$" , max_len: 36}];
+  map<string,string> attributes = 3 [
+    (validate.rules).map = {
+      max_pairs: 10,
```

Comment:
Why only 10?

Reply:
Like any other limit we set, it's pretty arbitrary. In our production databases we don't see any entities with more than 10 attributes. So let's turn the question around: Why more than 10?

Reply:
I guess for how we use attributes currently it's fine. I remember some discussion around using labels e.g. for tags, in which case I can see more than 10 attributes being beneficial. As long as we can increase the limit later on, it's fine I'd say.